### ISSUE TITLE
[Bug] Missing block particles

### ISSUE THREAD

**Issue by Nuanulla** [sp614x/optifine#6349]
## Description of Issue
Block particles from breaking blocks no longer appear when Optifine H1 pre8 is used as a mod with Forge. Issue doesn't occur when H1 pre8 isn't present. Also doesn't occur with past Optifine previews and versions.

## Steps to Reproduce
1. Make sure Optifine H1 pre8 is in the "mods" folder
2. Break a block (any block) in-game.

## OptiFine Version
1.17.1-Optifine_HD_U_H1_pre8

## Installation Method
Tried different methods: Plopped installer .jar into "mods" folder, separately extracted mod using installer, and installed new Optifine profile via installer.

## Forge Version
1.17.1-forge-37.0.103

## Other Installed Mods
Inapplicable as issue persists without presence of other mods. See below.

## Log Files/Crash Reports
Latest log: https://pastebin.com/ZcKBtcva
Debug log: https://pastebin.com/tQS29cVt

## F3 Debug Screenshot
https://imgur.com/a/VtWJvvz

## Prior Testing
- Does this happen in Vanilla without OptiFine?
No

If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
      Yes.
- Does this happen without any mods at all?
      No. Issue doesn't occur when playing in a Forge installation and without Optifine.
- Does this happen in OptiFine standalone?
      No.
- Does this happen in Vanilla without OptiFine?
      No. This is a repeated question.
- Have you tried using a binary search to find which mods cause this issue?
      N/A. This issue is caused when Forge is paired with Optifine. Unsure which of the two is the primary causer.
   - Does the issue still happen with only the remaining mods installed?
      Unsure what this question means. Please expand on template. Issue doesn't occur when all other Forge mods are present and without Optifine.

## Additional Information
To summarise or expand:
Issue persists with the following:
- new worlds/saves
- different installation methods
- JDK 16.0.2 and Minecraft's packaged 16.0.1
- without an active resource pack
- without any other active Forge mods
- default Optifine settings (removed optionsof.txt)

Issue does not occur when Optifine is installed as standalone (version and profile).

Other details:
- I'm playing on macOS 12.0.1 (Monterey)
- Forge is being used in a separate game directory (mods folder isn't located in the .minecraft folder). This shouldn't be an issue as the block particle issue isn't present when Optifine is installed and used in a separate directory as well.

**Comment by Nuanulla** [comment]
Issue doesn't occur with 1.17.1_HD_U_H1_pre7 on 1.17.1-forge-37.0.103 or 1.17.1-forge-37.0.104. There must be some sort of undocumented change in pre8. Otherwise, one of the "smooth lighting fixes" broke block particles.

**Comment by Nuanulla** [comment]
Issue persists with Optifine 1.17.1_HD_U_H1 on 1.17.1-forge-37.0.103, macOS 12.0.1, packaged JDK 16.0.1.

**Comment by Looxond** [comment]
Can confirm issue persists on macOS 11.6

**Comment by sp614x** [comment]
Fixed in H2_pre1
